lib/blockscore/collection/member.rb

```diff
@@ -0,0 +1,100 @@
+module BlockScore
+  class Collection
+    # Member of a {Collection} class
+    class Member < SimpleDelegator
+      # Initialize a new member
+      #
+      # @param parent [BlockScore::Base] parent resource
+      # @param instance [BlockScore::Base] member instance
+      #
+      # @return [undefined]
+      #
+      # @api private
+      def initialize(parent, instance)
+        @instance = instance
+        @parent = parent
+
+        super(instance)
+      end
+
+      # Save parent, set parent id, and save instance
+      #
+      # @example
+      #   # saves both unsaved person and unsaved question_set
+      #   person = Person.new(attributes)
+      #   question_set = QuestionSet.new
+      #   Member.new(person, question_set).save
+      #
+      # @return return value of instance `#save`
+      #
+      # @api public
+      def save
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for save is too high. [15.07/15]